src/4.3_to_5.0/_addDefaultMeasureIfNoneIsExplicitlyExpressed.test.ts

```diff
@@ -0,0 +1,44 @@
+import { MdxSelect, parse, stringify } from "@activeviam/mdx-5.0";
+import { _addDefaultMeasureIfNoneIsExplicitlyExpressed } from "./_addDefaultMeasureIfNoneIsExplicitlyExpressed";
+import { dataModelsForTests } from "@activeviam/data-model-5.0";
+
+const cube = dataModelsForTests.sandbox.catalogs[0].cubes[0];
+
+describe("_addDefaultMeasureIfNoneIsExplicitlyExpressed", () => {
```

Comment:
It would be nice to add a test ensuring that the function does not early-return by mistake when the query does not contain any measure but a hierarchy is sorted. For instance:

```
WITH
 Member [Measures].[[Currency]].[Currency]].[Currency]]_for_order] AS Rank(
  [Currency].[Currency].CurrentMember,
  [Currency].[Currency].Members
) 
SELECT
  NON EMPTY Order(
    Hierarchize(
      Descendants(
        {
          [Currency].[Currency].[ALL].[AllMember]
        },
        1,
        SELF_AND_BEFORE
      )
    ),
    [Measures].[[Currency]].[Currency]].[Currency]]_for_order],
    DESC
  ) ON ROWS
  FROM [EquityDerivativesCube]
```

I expect `[Measures].[contributors.COUNT]` to be added in this query.